guides/user/kubernetes-v2/rollout-strategies/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: single
+title:  "Rollout Strategies"
+sidebar:
+  nav: guides
+---
+
+{% include alpha version="1.14" %}
+
+{% include toc %}
+
+This guide describes how to take advantage of the
+[Kubernetes V2](/setup/install/providers/kubernetes-v2) provider's first-class support
+for common rollout strategies, including dark, highlander, and red/black rollouts.
+
+The implementation of these rollout strategies currently leverages Spinnaker's existing
+[traffic management strategy](/guides/user/kubernetes-v2/traffic-management/) and so is
+[valid for ReplicaSets only](/guides/user/kubernetes-v2/traffic-management/#you-must-use-replica-sets).
```

Comment:
I'm not sure if we mention this elsewhere but it would be great if we could call out exactly why advanced traffic management isn't possible with Deployments. It seems like it might be a fairly common question.

Reply:
Agreed, the `valid for ReplicaSets only` text links [here](https://www.spinnaker.io/guides/user/kubernetes-v2/traffic-management/#you-must-use-replica-sets) which explains "why not deployments", lmk if you think that's enough or we should highlight it more somehow?